compiler/src/pretyping.ml

```diff
@@ -272,6 +298,7 @@ end  = struct
     } 
 
   type 'asm global_bindings = {
+      gb_types : (A.symbol, P.pty L.located) Map.t;
```

Comment:
Is the location needed here?

Reply:
Yes it is for producing more precise error (with correct location from duplicated variables)